packages/website/ts/pages/governance/governance.tsx

```diff
@@ -0,0 +1,302 @@
+    2: 'Medium Benefit',
+    3: 'High Benefit',
+};
+
+const riskLabels: LabelInterface = {
+    1: 'Low Risk',
+    2: 'Medium Risk',
+    3: 'High Risk',
+};
+
+const proposalData = {
+    zeipId: 1,
+    title: 'MultiAssetProxy Approval',
+    summary: `MultiAssetProxy brings support for trading arbitrary bundles of assets in 0x protocol. Historically only a single asset could be traded per each side of a trade. With the introduction of the MultiAssetProxy, users will be able to trade multiple ERC721 assets and even mix ERC721 and ERC20 to trade in a single order.`,
+    url: '#',
+    votingDeadline: 1551584000,
+    benefit: {
+        title: 'Ecosystem Benefit',
+        summary: `Supporting bundled trades is one of the most commonly requested features since the launch of 0x v2. Demand originated from our discussions with gaming and NFT related projects, but this upgrade provides utility to Prediction Markets as well as any relayer that wants to offer baskets of tokens. The MultiAssetProxy will enable new possibilities of trading.  *High benefit 4/5*`,
+        rating: 3,
+        links: [
+            {
+                text: 'Learn More',
+                url: 'https://0xproject.quip.com/k1ERAXnUj2ay/ZEIP23-Support-for-MultiAssetProxy',
+            },
+            {
+                text: 'Technical detail',
+                url: 'https://github.com/0xProject/ZEIPs/issues/23',
+            },
+        ],
+    },
+    stakes: {
+        title: 'Risk',
+        summary: `Deploying the MultiAssetProxy is a hot upgrade to 0x protocol where the state of active contracts is modified. The contracts being modified contain allowances to users tokens. As such the MultiAssetProxy has successfully undergone a third party audit. We encourage the community to verify the MultiAssetProxy code along with the state changes.`,
+        rating: 2,
+        links: [
+            {
+                text: 'View Code',
+                url:
+                    'https://github.com/0xProject/0x-monorepo/blob/development/contracts/asset-proxy/contracts/src/MultiAssetProxy.sol#L25',
+            },
+            {
+                text: 'View Audit',
+                url: 'https://github.com/ConsenSys/0x-audit-report-2018-12',
+            },
+        ],
+    },
+    complexity: {
+        title: 'Complexity of Code',
+        summary: `The world's assets are becoming tokenized on public blockchains. 0x Protocol is free, open-source infrastracture that developers and businesses utilize to build products that enable the purchasing and trading of crypto tokens.`,
+        rating: 1,
+        links: [
+            {
+                text: 'View Code',
+                url: '#',
+            },
+        ],
+    },
+};
+
+export class Governance extends React.Component {
+    public state: State = {
+        isContactModalOpen: false,
+        isVoteModalOpen: false,
+        isWalletConnected: false,
+        isVoteReceived: false,
+        providerName: 'Metamask',
```

Comment:
Website has a utility function some where that detects the underlying Wallet where possible. For example on Coinbase Wallet this will mention Metamask which is confusing.